crates/re_data_store/src/store.rs

```diff
@@ -583,98 +542,40 @@ impl Default for IndexedBucketInner {
     }
 }
 
-// --- Timeless ---
+// --- Static ---
 
-/// The timeless specialization of an [`IndexedTable`].
-///
-/// Run the following command to display a visualization of the store's internal datastructures and
-/// better understand how everything fits together:
-/// ```text
-/// cargo test -p re_data_store -- --nocapture datastore_internal_repr
-/// ```
-#[derive(Debug)]
-pub struct PersistentIndexedTable {
+/// Keeps track of static component data per entity.
+#[derive(Clone)]
+pub struct StaticTable {
     /// The entity this table is related to, for debugging purposes.
-    pub ent_path: EntityPath,
+    pub entity_path: EntityPath,
 
     /// Carrying the cluster key around to help with assertions and sanity checks all over the
     /// place.
     pub cluster_key: ComponentName,
 
-    // To simplify interior mutability.
-    pub inner: RwLock<PersistentIndexedTableInner>,
-}
-
-impl Clone for PersistentIndexedTable {
-    fn clone(&self) -> Self {
-        Self {
-            ent_path: self.ent_path.clone(),
-            cluster_key: self.cluster_key,
-            inner: RwLock::new(self.inner.read().clone()),
-        }
-    }
+    pub cells: BTreeMap<ComponentName, StaticCell>,
```

Comment:
What are the semantics of subsequent writes? Latest wins? Highest row_id wins? Please document here 🙏